inspirehep/modules/migrator/cli.py

```diff
@@ -54,6 +57,8 @@
 def migrator():
     """Command related to migrating INSPIRE data."""
     logging.basicConfig()
+    # Disable auto-indexing receiver in migration tasks
+    models_committed.disconnect(receive_after_model_commit)
```

Comment:
To be on the safe side I am reconnecting the signal https://github.com/inspirehep/inspire-next/pull/1775/files#diff-7e8feab1a9e8ec565a17aca30402e654R66

If Travis passes it is ready to be merged.